mpi_ring_c_test.sh

```diff
@@ -26,21 +28,32 @@ fi
 
 host_setup ${hostfile} ${hosts}
 mpicc -o /tmp/ring_c ring_c.c
+# Fortran compile test
+mpif90 -o /tmp/ring_usempi ring_usempi.f90
```

Comment:
Would it make sense to have a separate file ```mpi_ring_fortran_test.sh```?

Reply:
I think so, I can update it

Reply:
I'll probably hold off on this though for now since it's not an important change, I'll follow up later on this

Reply:
Thanks, would you please set the name of the binary to ```ring_fortran```. The current name ```ring_usempi``` is a little confusing.

Reply:
Sure